src/map/lapack2flamec/f2c/scripts/regen-files.sh

```diff
@@ -5,16 +5,24 @@ tar_file=$1
 
 ## Clean up all
 rm -f ../c/*.c ../c/*~
-rm -f ../netlib/*.f ../netlib/*~
+rm -f ../netlib/*.f ../netlib/*.F ../netlib/*.f ../netlib/*.F90 ../netlib/*~
+rm -rf ../netlib/lapack-[0-9]*.[0-9]*.[0-9]*/
 
 ## 
 if [[ "$tar_file" == *.tgz || "$tar_file" == *.tar.gz ]]
 then
-    (cd ../netlib; tar -xzvf $tar_file; rm -f ._*.f; cat support.list | xargs rm -f; cat install.list | xargs rm -f)
+    (cd ../netlib; tar -xzf $tar_file ;
+    mv lapack-[0-9]*.[0-9]*.[0-9]*/SRC/*.f .;
+    rm -f ._*.f;
+    cat support.list | xargs rm -f;
+    cat install.list | xargs rm -f)
 elif [[ "$tar_file" == *.tar ]]
 then
-    ## xerbla.f and any .f using f90 intrinsic (maxloc) are not imported
-    (cd ../netlib; tar -xvf $tar_file; rm -f ._*.f; cat support.list | xargs rm -f; cat install.list | xargs rm -f)
```

Comment:
Left in support for uncompressed tar's. However, using tgz will save a lot of space.